docs/UserGuide.md

```diff
@@ -324,12 +325,17 @@ Volunteer Roles (Code - Role Name):
 
 ## Command summary
 
-Action | Format, Examples
---------|------------------
-**Add** | `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​` <br> e.g., `add n/James Ho p/22224444 e/jamesho@example.com a/123, Clementi Rd, 1234665 t/friend t/colleague`
-**Clear** | `clear`
-**Delete** | `delete INDEX`<br> e.g., `delete 3`
-**Edit** | `edit INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`<br> e.g.,`edit 2 n/James Lee e/jameslee@example.com`
-**Find** | `find KEYWORD [MORE_KEYWORDS]`<br> e.g., `find James Jake`
-**List** | `list`
-**Help** | `help`
+| Action                | Format, Examples                                                                               |
+|-----------------------|------------------------------------------------------------------------------------------------|
+| **Add Person**        | `add n/NAME p/PHONE_NUMBER m/EMAIL [r/ROLE]…​`                                                 |
+| **Add Event**         | `addevent sp/SPORT t/Faculty 1 t/Faculty 2 d/LocalDateTime v/Venue [pa/PARTICIPANTS`           |
```

Comment:
I think the capitalisation for placeholders behind prefixes should be fixed to all caps (e.g. t/FACULTY 1, t/FACULTY 2, etc.)